src/override/malloc.cc

```diff
@@ -205,6 +205,17 @@ extern "C"
     return ENOENT;
   }
 
+  SNMALLOC_EXPORT void* SNMALLOC_NAME_MANGLE(get_global_pagemap)(void)
```

Comment:
I wonder if these should be conditionally compiled?  `#ifdef` As we don't need these symbols in most scenarios.
 E.g.
```C
#ifdef SNMALLOC_EXPOSE_PAGEMAP
```
and
```C
#ifdef SNMALLOC_EXPOSE_RESERVE
```

Reply:
I would also like to see this one prefixed with snmalloc.  We probably do want to support this in the libc implementation.  I had a FIXME somewhere to expose something like this, but taking the version (or possibly some encoding of the configuration) so that we can return null if there's a mismatch between the pagemap representation that the caller expects and the callee provides.  Since we're carefully sidestepping type safety here, we need to reintroduce it somehow.